src/server.h

```diff
@@ -485,7 +485,7 @@ class Server : public con::PeerHandler, public MapEventReceiver,
 	void SendHP(session_t peer_id, u16 hp, bool effect);
 	void SendBreath(session_t peer_id, u16 breath);
 	void SendAccessDenied(session_t peer_id, AccessDeniedCode reason,
-		const std::string &custom_reason, bool reconnect = false);
+		std::string_view custom_reason, bool reconnect = false);
```

Comment:
`DenyAccess` should take a string view too

Reply:
Done in 304a89e34